relay-server/src/actors/processor/report.rs

```diff
@@ -469,26 +469,15 @@ mod tests {
             item
         });
 
-        envelope.add_item({
-            let mut item = Item::new(ItemType::Event);
-            item.set_payload(ContentType::Json, "{}");
-            item
-        });
```

Comment:
Are we explicitly disallowing envelopes that contain an event and a user report now?

Reply:
Right now the processor will always get those items in the separate envelopes. Since we split the envelope in the endpoint. I can also move the split back into processor, and this test will still look as it did before.